config/database.yml

```diff
@@ -17,11 +17,21 @@
 development: &default
   adapter: postgresql
   encoding: unicode
-  database: champaign
+  database: champaign_test
```

Comment:
I'll change it it to `champagin` over at `champaign-ak-processor` so we can change it back here. Otherwise, all development data will get purged when we run tests.